src/app/files/shared-folder.js

```diff
@@ -45,6 +45,10 @@ class SharedFolder {
     cb()
   }
 
+  checkSpecialChars (name) {
+    return name.match(/(\/|:|\*|\?|"|<|>|\\|\||')/) != null
```

Comment:
Can you create this function only once?

Reply:
Ah and something like `/[/:*?"<>\\\|]/` would be better I think.

Reply:
one is for browser-storage and the other for shared-folder (remixd). there might have a different requirement for `checkSpecialChars` ... but I guess that is fine to keep the same behavior for both